### ISSUE TITLE
Aporeto: Validate OCP Upgrade

### ISSUE THREAD

**Issue by j-pye** [BCDevOps/OpenShift4-RollOut#249]
During the OCP 4.3.18 -> OCP 4.4.4 Upgrade we experienced an issue with a  MutatingWebhookConfiguration blocking the network operator from upgrading. Aporeto was installed but disabled at the time. After deleting the Aporeto operator the installation continued as expected. 

- [x] Verify Aporeto Operator will not block OCP Upgrades
- [x] Verify Aporeto Enforcers will not block OCP Upgrades

**Comment by j-pye** [comment]
I need another cluster to test this on. I say this because there's potential to stall all work on said cluster during testing. Open to suggestions for testing this sooner.

**Comment by tosazuwa** [comment]
@j-pye , @sbarre-esit to update with testing results/concerns

**Comment by j-pye** [comment]
During the OCP 4.4.11 upgrade we experienced issues relating to the Aporeto Network Access Policies. These issues did not block the upgrade, however, an upgrade should be tested with all enforcers enabled before closing this ticket.

All OCP Cluster Operators upgraded successfully during this upgrade.

**Comment by j-pye** [comment]
September 23, 2020
- KLAB was upgraded to OCP 4.5.11
- Aporeto Enforcement was enabled on all nodes
- Prior to the upgrade we discovered the Router Metrics were failing and that there was a missing policy. This did not impact the upgrade.
- During our previous upgrade on September 18th, we discovered that Mutating/Admission webhooks directed at pods can cause an upgrade to fail if the webhook endpoints are not reachable so those services must be up and accessible. (In this instance the failure was caused by a missing policy which stopped the upgrade job pod from being run)
- During the upgrade Elastic Search went into a RED state but eventually recovered on its own
